taxonomies/additives.txt

```diff
@@ -1592,6 +1598,7 @@ efsa_evaluation:en: Scientific Opinion on the reconsideration of the ADI and a r
 vegan:en:maybe
 vegetarian:en:maybe
 
+<en:E160
```

Comment:
For the a/b/c/etc. additives, like E160a, we do not want to add E160 as a parent, it doesn't really exist and it will mess the additives detection.